docs/root/start/start.rst

```diff
@@ -0,0 +1,154 @@
+.. _start:
+
+Getting Started
+===============
+
+This section gets you started with a very simple configuration, and provides some example configurations.
+
+Envoy does not currently provide separate pre-built binaries, but does provide Docker images. This is
+the fastest way to get started using Envoy. Should you wish to use Envoy outside of a
+Docker container, you will need to :ref:`build it <building>`.
+
+These examples use the :ref:`v2 Envoy API <envoy_api_reference>`, but use only the static configuration
+feature of the API, which is most useful for simple requirements. For more complex requirements
+:ref:`Dynamic Configuration <arch_overview_dynamic_config>` is supported.
+
+Quick Start to Run Simple Example
+---------------------------------
+
+These instructions run from files in the Envoy repo. The sections below give a
+more detailed explanation of the configuration file and execution steps for
+the same configuration.
+
+A very minimal Envoy configuration that can be used to validate basic plain HTTP
+proxying is available in :repo:`configs/google_com_proxy.v2.yaml`. This is not
+intended to represent a realistic Envoy deployment.
+
+Copy both :repo:`configs/Dockerfile` and
+:repo:`configs/google_com_proxy.v2.yaml` to the same directory on your local
+disk. Then, build and run the Dockerfile, and test out Envoy by sending a
+request to port 10000::
+
+  $ docker build -t envoy-google-test:v1 .
+  $ docker run -d -p 10000:10000 envoy-google-test:v1
```

Comment:
Nit: is there a link in here to docker and how to set it up locally?

Reply:
And can each command be commented

Reply:
The each command is separated and commented on in the section below. This section was essentially the older quickstart (and I added a paragraph in the front to say the explanation is given below), so I think this is OK (would you agree?).  I will add the docker link though.

Reply:
On second thought, in general I don't see documentation linking to Docker instructions, as they are outside of the scope of the documentation, and we should not have links to places where things could change. So I think we can assume people know how to deal with Docker and if they can't, they are able to operate teh google to figure it out.